api/admin/routes.py

```diff
@@ -411,6 +411,22 @@ def patron_auth_services():
 def patron_auth_service(service_id):
     return app.manager.admin_settings_controller.patron_auth_service(service_id)
 
+@app.route("/admin/manage_patrons", methods=['GET', 'POST'])
+@has_library
```

Comment:
The `has_library` decorator only works if the library short name is in the route (e.g. if you're also using `library_route`). 

Also, I don't think either of these routes needs to allow GETs.